apps/checkout-app/pages/api/webhooks/adyen.ts

```diff
@@ -102,7 +102,7 @@ export default async function handler(
       adyen.hmac!
     );
   } catch (error) {
-    return res.status(401).send(error);
+    return res.status(401).send(error instanceof Error ? error.message : String(error));
```

Comment:
I don't think we should even return a caught error message here, as CodeQL suggested. The stack trace could include some sensitive information (like for example Adyen API keys). I think a generic error message + logging would be better:

```suggestion
    console.error(error);
    return res.status(401).send("Error while handling webhook");
```